assets/queries/cloudFormation/iot_policy_allows_resource/test/positive.yaml

```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+    IoTPolicy:
+      Type: AWS::IoT::Policy
+      Properties:
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+            - iot:Connect
+            Resource: "*"
+          - Effect: Deny
+            Action:
+            - sqs:*
+            NotResource:
+            - !GetAtt myqueue.Arn
```

Comment:
Let's get rid of this `GetAtt` function since the referenced Resource `myqueue` does not exist in this template, just use:

`NotResource: my-hardcoded-arn`